lib/common/ipc_pacemakerd.c

```diff
@@ -208,6 +208,17 @@ dispatch(pcmk_ipc_api_t *api, xmlNode *reply)
         return ack_status == CRM_EX_INDETERMINATE;
     }
 
+    value = crm_element_value(reply, PCMK__XA_T);
+    if (pcmk__parse_server(value) != pcmk_ipc_pacemakerd) {
+        /* @COMPAT pacemakerd <3.0.0 sets PCMK__VALUE_CRMD as the message type,
```

Comment:
ah just bump the remote protocol version and no more worrying about newer remote nodes :)

actually, "remote clients" in the commit message might mean cluster nodes connecting to the remote server, or remote nodes that are clients to the cluster node's pacemakerd, I don't know. I've never fully understood the remote daemon proxying.